.changeset/rotten-gifts-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+"claude-dev": minor
+---
+
+Add notification system and active conversation features for external MCP tools integrationThis changeset introduces two key features that enable seamless integration between Cline and external MCP tools like Claude Desktop:1. Notification System Bridge: - Allows external MCP tools to send messages, suggestions, and advice to active Cline conversations - Implements a notification panel with read, dismiss, and restore functionality - Provides a "Send to Chat" feature to easily incorporate external advice into conversations2. Active Conversation Marking: - Enables users to mark conversations as "Active A" or "Active B" - Allows external MCP tools to detect and access these conversations - Creates a seamless workflow between Cline and other AI assistantsThese features are implemented using the cline-mcp-tools MCP server (https://github.com/anthonyjj89/cline-mcp-tools) and have been tested with Claude Desktop.This integration creates a cost-effective workflow where users can use Claude Desktop for research and planning, then send targeted advice to Cline without incurring per-message costs.Note: Currently, the integration has only been tested with Claude Desktop. Future enhancements will extend support to other external MCP tools.
```

Comment:
This is going to be a VERY long entry for our changeset automation.  Can you summarize this to about 1 or 2 sentences?